lua/textcase/plugin/plugin.lua

```diff
@@ -155,12 +155,16 @@ function M.incremental_substitute(opts, preview_ns, preview_buf)
 
       local get_match = utils.get_list(utils.escape_string(transformed_source), mode)
       for match in get_match do
-        if dest ~= "" then
-          conversion.replace_matches(match, transformed_source, transformed_dest, false, buf)
-        end
-        local length = transformed_dest == "" and #transformed_source or #transformed_dest
-        if preview_ns ~= nil then
-          vim.api.nvim_buf_add_highlight(buf, preview_ns, "Search", match[1] - 1, match[2] - 1, match[2] - 1 + length)
+        local match_is_inside_visual_range = match[1] >= opts.line1 and match[1] <= opts.line2
```

Comment:
It fixes https://github.com/johmsalas/text-case.nvim/issues/109 but I didn't fully grok how the Subs command is working.

One issue is that if you use `<CMD>` instead `:` in the test, it won't work because `opts.line1` and `opts.line2` are then first and last line of the file somehow.

Reply:
Probably because `:` in vim adds `'<,'>` while `<CMD>` does not. Which means this case can be ignored.

Reply:
> I didn't fully grok how the Subs command is working

Hope this helps https://github.com/johmsalas/text-case.nvim/pull/112

Reply:
Thanks a lot :)